tests/pkg_flashdb_mtd/Makefile

```diff
@@ -0,0 +1,21 @@
+include ../Makefile.tests_common
+
+# select the MTD backend
+USEMODULE += flashdb_mtd
+# enable time series database
+USEMODULE += flashdb_tsdb
+# enable key-value database
+USEMODULE += flashdb_kvdb
+
+# the test makes use of time()
+USEMODULE += libc_gettimeofday
+
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=THREAD_STACKSIZE_LARGE
+
+CFLAGS += -DFAL_PART0_LABEL=\"/fdb_kvdb1\"
+CFLAGS += -DFAL_PART0_LENGTH=8*1024
+
+CFLAGS += -DFAL_PART1_LABEL=\"/fdb_tsdb1\"
+CFLAGS += -DFAL_PART1_LENGTH=8*1024
```

Comment:
Any reasoning for these lengths?

Reply:
Then is it possible to make these values configurable somewhere ? Maybe in the package Makefile.include or something like this that works.

Reply:
Sorry, I didn't notice those were set in the application Makefile. So good for me like this :)